src/MLJGLMInterface.jl

```diff
@@ -290,6 +290,15 @@ function glm_report(glm_model, features, reportkeys)
     return NamedTuple{Tuple(keys(report_dict))}(values(report_dict))
 end
 
+function _ensure_intercept_at_end(form::FormulaTerm)
+    fixed_rhs = if first(form.rhs) isa ConstantTerm
+        (form.rhs[2:end]..., form.rhs[1])
+    else
```

Comment:
yeah I think this is probably a reasonable workaround.  I will call out that there's a possibility that the internal format for how multiple terms are represented may change (specifically, I'd like to switch to a container that is not as "strongly typed" as a tuple, like a vector).  I don't think that the ability to construct a tuple will ever go away but it might come with performance implications (mostly in the form of unnecessary compilation/specialization which is the current situation).

I don't think there's anything that you need to DO about this right now, just wanted to raise awareness!